test/e2e_node/util.go

```diff
@@ -82,7 +85,7 @@ func getNodeSummary() (*stats.Summary, error) {
 	if err != nil {
 		return nil, fmt.Errorf("failed to get current kubelet config")
 	}
-	req, err := http.NewRequest("GET", fmt.Sprintf("http://%s:%d/stats/summary", kubeletConfig.Address, kubeletConfig.ReadOnlyPort), nil)
+	req, err := http.NewRequest("GET", fmt.Sprintf("http://%s/stats/summary", net.JoinHostPort(kubeletConfig.Address, strconv.Itoa(int(kubeletConfig.ReadOnlyPort)))), nil)
```

Comment:
why so many conversions from int32 to string? It looks like [JoinHostPort](https://pkg.go.dev/vitess.io/vitess/go/netutil#JoinHostPort) can take int32 as an argument and `kubeletConfig.ReadOnlyPort` is `int32` already

Reply:
You're looking at a version of JoinHostPort that is not part of the standard library.

This way of doing things is already common in this codebase (and many others):

```
$ grep -r JoinHostPort . | grep strconv | wc -l
89
```

Reply:
got it. Indeed my google search failed me.